operatorconfig/moduleconfig/authorization/v1.10.0/deployment.yaml

```diff
@@ -302,91 +381,181 @@ spec:
+            else
+              echo "Sentinel found, finding master"
+              MASTER="$(redis-cli -h sentinel -p 5000 sentinel get-master-addr-by-name mymaster | grep -E '(^redis-csm-\d{1,})|([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})')"              
+              echo "replicaof $MASTER_FDQN 6379" >> /etc/redis/redis.conf 
+            fi
+        volumeMounts:
+        - name: redis-primary-volume
+          mountPath: /data
+        - name: configmap
+          mountPath: /csm-auth-redis-cm/
+        - name: config
+          mountPath: /etc/redis/
       containers:
-      - name: primary
+      - name: redis-csm
         image: <AUTHORIZATION_REDIS_IMAGE>
-        imagePullPolicy: IfNotPresent
-        args: ["--appendonly", "yes", "--appendfsync", "always"]
-        resources:
-          requests:
-            cpu: 100m
-            memory: 100Mi
+        command: [ "redis-server" ]
+        args: [ "/etc/redis/redis.conf" ]
         ports:
         - containerPort: 6379
+          name: <AUTHORIZATION_REDIS_NAME>
         volumeMounts:
-          - name: redis-primary-volume
-            mountPath: /data
-      volumes:
         - name: redis-primary-volume
-          persistentVolumeClaim:
-            claimName: redis-primary-pv-claim
----
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: redis-primary-pv-claim
-  namespace: <NAMESPACE>
-  labels:
-    app: redis-primary
-spec:
-  accessModes:
-    - ReadWriteOnce
-  storageClassName: <REDIS_STORAGE_CLASS>
-  resources:
-    requests:
-      storage: 8Gi
+          mountPath: /data
+        - name: configmap
+          mountPath: /csm-auth-redis-cm/
+        - name: config
+          mountPath: /etc/redis/
+      volumes:
+      - name: redis-primary-volume
+        emptyDir: {}
+      - name: config
+        emptyDir: {}
+      - name: configmap
+        configMap:
+          name: redis-csm-cm
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: redis-commander
+  name: <AUTHORIZATION_REDIS_COMMANDER>
   namespace: <NAMESPACE>
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: redis-commander
+      app: <AUTHORIZATION_REDIS_COMMANDER>
   template:
     metadata:
       labels:
         csm: <NAME>
-        app: redis-commander
+        app: <AUTHORIZATION_REDIS_COMMANDER>
         tier: backend
     spec:
       containers:
-      - name: redis-commander
+      - name: <AUTHORIZATION_REDIS_COMMANDER>
         image: <AUTHORIZATION_REDIS_COMMANDER_IMAGE>
         imagePullPolicy: IfNotPresent
         env:
-        - name: REDIS_HOSTS
-          value: "rbac:redis.<NAMESPACE>.svc.cluster.local:6379"
+        - name: SENTINELS
+          value: "sentinel-0.sentinel.<NAMESPACE>.svc.cluster.local:5000, sentinel-1.sentinel.<NAMESPACE>.svc.cluster.local:5000, sentinel-2.sentinel.<NAMESPACE>.svc.cluster.local:5000, sentinel-3.sentinel.<NAMESPACE>.svc.cluster.local:5000, sentinel-4.sentinel.<NAMESPACE>.svc.cluster.local:5000"
```

Comment:
Let us see if we can compute the names of sentinel dynamically.

Reply:
Made the changes. Finding the sentinel values using go code and passing the value to the yaml.